scene/gui/gradient_edit.cpp

```diff
@@ -236,8 +236,8 @@ void GradientEdit::_gui_input(const Ref<InputEvent> &p_event) {
 
 		//Snap to nearest point if holding shift
 		if (mm->get_shift()) {
-			float snap_treshhold = 0.03;
-			float smallest_ofs = snap_treshhold;
+			float snap_threshold = 0.03;
+			float smallest_ofs = snap_threshold;
```

Comment:
this is a little wiered anyways. `snap_thershold` does only get used there. 
maybe just do.
```
float smallest_ofs = 0.3; \\ acts as the snap threshold
```
or just rename `smallest_ofs` to `snap_threshold`

Reply:
There is something like : if (bla bla bla) { smallest_ofs = temp_ofs; ...... }

So, I'm not sure. ;-)